server/rest/src/main/java/org/infinispan/rest/configuration/RestServerConfiguration.java

```diff
@@ -49,8 +52,16 @@ public int maxContentLength() {
       return maxContentLength;
    }
 
+   /**
+    * @deprecated Use {@link #getCorsRules()} instead.
+    */
+   @Deprecated
```

Comment:
Doesn't this mean that we can't include this in 9.2.1.Final?

Reply:
This should not be @deprecated, as it was added after 9.2. Force of habit :smile: